plugins/operator.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: operator
+spec:
+  version: v0.0.3
+  homepage: https://github.com/operator-framework/kubectl-operator
+  shortDescription: Manage Kubernetes Operators with Operator Lifecycle Manager
+  description: |
+    This plugin is a package manager for Operators in your cluster. It
+    simplifies adding and removing Operator catalogs, and it has familiar
+    commands for installing, uninstalling, and listing available and
+    installed Operators.
```

Comment:
IIUC, the plugin in particular facilitates the usage of operators from https://operatorhub.io/. Might be worth mentioning.

Reply:
@corneliusweig While that is true operatorhub.io is merely just a public index. Users of that toolchain have the ability to create their own indexes and the `kubectl operator` plugin allows to use those as well. So there could be a pointer somewhere to operatorhub.io as an example for a catalog to add to your cluster, but otherwise they are reasonably decoupled.